bigtable/tests/unit/test_table.py

```diff
@@ -1482,16 +1593,24 @@ def test_do_mutate_retryable_rows_second_try_no_retryable(self):
         row_2 = DirectRow(row_key=b"row_key_2", table=table)
         row_2.set_cell("cf", b"col", b"value2")
 
-        worker = self._make_worker(client, table.name, [row_1, row_2])
-        worker.responses_statuses = self._make_responses_statuses(
-            [self.SUCCESS, self.NON_RETRYABLE]
-        )
-
-        statuses = worker._do_mutate_retryable_rows()
+        row_3 = DirectRow(row_key=b"row_key3", table=table)
+        row_3.set_cell("cf", b"col", b"value3")
+        row_4 = DirectRow(row_key=b"row_key_4", table=table)
+        row_4.set_cell("cf", b"col", b"value4")
```

Comment:
Since the same row-generating sequence is repeated multiple times within a single file, maybe it makes sense to encapsulate it in a 'private' helper method.